packages/api-explorer/__tests__/index.test.jsx

```diff
@@ -417,13 +400,8 @@ describe('stripSlash', () => {
 })
 
 describe('forcePanelRender', () => {
-  it('should default to false', () => {
```

Comment:
Also this test

Reply:
it's a little useful test. let's see if the prop we pass to the component has the value just given